documentation/modules/ROOT/pages/configuration/mongodb-event-flattening.adoc

```diff
@@ -1,5 +1,6 @@
+[id="mongodb-new-document-state-extraction"]
 = MongoDB New Document State Extraction
```

Comment:
On around Line 185, in the `MongoDB $unset handling` section, there is a reference to Debezium.